Lib/test/test_userlist.py

```diff
@@ -11,6 +11,16 @@ class UserListTest(list_tests.CommonTest):
     def test_init(self):
         super().test_init()
```

Comment:
The `test_init` method in `list_tests.CommonTest` is no longer marked. I don't think this needs to be here.

Reply:
@fanninpm  we expect code review is about changed code, instead of any code around changed code.

I think that change is independent from this PR. If you want me to fix it, I will do it in separated PR.